packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -23,6 +23,23 @@ export interface IService extends IResource {
   readonly serviceArn: string;
 }
 
+export interface EcsTarget {
+  /**
+   * Properties for defining an ECS target.
+   */
+  readonly containerTarget: LoadBalancerTargetOptions,
```

Comment:
All other interfaces are declared using a semicolon (`;`) at the end of fields, so please match the existing style.

Reply:
Also, I think I would be in favor of inlining the `LoadBalancerTargetOptions` property into the parent struct here. The usability will be a lot better in non-JS languages.